server/data/dev-data/questions.json

```diff
@@ -64,5 +64,27 @@
 				"downvotes": 1
 			}
 		]
+	},
+	{
+		"userQuestioner": "Mrunaal Chincholikar",
+		"questionTitle": "This is a question 4",
+		"questionDescription": "This is the description of the question",
+		"questionTag": "Courses",
```

Comment:
btw this `https://github.com/kaiwalyakoparkar/quora-for-college/blob/main/server/models/questionsModel.js#L17` has all the allowed question tags. Could you please change 'Courses' to something different?